### ISSUE TITLE
XONSH_STORE_STDOUT limits

### ISSUE THREAD

**Issue by anki-code** [con-f-use/xontrib-histcpy#14]
Hi! Thank you for this histcpy!

1. When `XONSH_STORE_STDOUT=True` is it has any limits to read output? If I do `cat gigabyte_file` it saves all gigbyte of data? If yes is it has any other risks?

2. Is there a way to avoid setting `XONSH_STORE_STDOUT` to `True`? For example get only latest output with events/hooks.

Thanks!

**Comment by anki-code** [comment]
Ask here https://github.com/xonsh/xonsh/issues/3534

**Comment by con-f-use** [comment]
Yeah, I think that is a good idea for your [output search](https://github.com/anki-code/xontrib-output-search) and should be a fall back or customization for histcopy. I just wanted to have more than the output of the last command and wanted it to work in a new terminal, although I could achieve the former with a rolling buffer, saving a couple of postcommands. I'll definitely think about it.

**Comment by anki-code** [comment]
The output search xontrib turned out very consistent and universal. If you want we can think how to make the histcpy features into it. 

For example:
* Add URL/IP tokenizer to have guarantee the it will be found clearly
* Add customazible hotkeys/commands support, for examle `Alt+U` to get URLs list
* Add `cpy` alias and when you need to copy url you can type `cpy <Alt+U>`
